### PR TITLE
Added hydrogenised cereals

### PR DESCRIPTION
**Pull request by aleene**
**Description:**

Added hydrogenised cereals

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -26280,13 +26286,14 @@ nl:gemalen maïs
...
+################################################################################################################
 #
-#                         Rice
+#                                                 R I C E
 #
-#######################################################################################################################################
+#################################################################################################################
 # Rice is the seed of the grass species Oryza sativa (Asian rice) or Oryza glaberrima (African rice).
 
+#<en:cereal
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt:26296]: why is it commented?

File: taxonomies/ingredients_processing.txt [hunk 2]
```diff
@@ -1135,3 +1135,13 @@ de:aus Früchten
...
 #to be analysed:
 #fr:rectifié
+
+#######################################################################
+#
+#                U L T R A - P R O C E S S I N G   P R O C E S S E S
+#
+#######################################################################
+
+en:hydrolysed
```
Comment (stephanegigandet) [inline taxonomies/ingredients_processing.txt:1145]: can we do that in a separate PR?

we would need corresponding code changes so that NOVA 4 ingredient processing are used when computing NOVA groups.

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (aleene) [reply taxonomies/ingredients.txt:26296]: It was not in there. So I wondered whether it should/can be added.

File: taxonomies/ingredients_processing.txt (hunk 2 above)
Comment (aleene) [reply taxonomies/ingredients_processing.txt:1145]: I'll comment it out for the moment.

**Comment by aleene** [comment]
What are these branch conflicts. I see no remarks/issues in the files.

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (stephanegigandet) [reply taxonomies/ingredients.txt:26296]: well if rice is a cereal, then I guess so.

**Comment by stephanegigandet** [comment]
> What are these branch conflicts. I see no remarks/issues in the files.

Someone else had edited the taxonomies, I fixed the conflicts.

**Review by stephanegigandet — APPROVED** [review]
Thanks!
